### PR TITLE
subject normalization

### PR DESCRIPTION
**Pull request by walkerdb**
Lots of subject cleanup.

Two main commits:
1. one from the normal open-refine clustering cleanup
2. one from a new set of scripts that does the following:
   1. Find all authfilenumbers that have multiple unique tag texts associated with them (like names with slightly different formatting) and export the results to a csv ([script](https://github.com/walkerdb/bentley_code/blob/master/one-off%20scripts/subject_manipulation/group_by_auth_id.py))
   2. Normalize all the examples that came up into a single form, based on data in a [manually-edited input csv](https://docs.google.com/spreadsheets/d/1-bn1sowmcsOdbyWSyzLz59ai1MpVEPQnZ7ZbtA9QFmg/edit#gid=0) based off of the output from the last step ([script](https://github.com/walkerdb/bentley_code/blob/master/one-off%20scripts/subject_manipulation/normalize_auth_ids_with_mult_texts.py))

### CODE REVIEW

File: Real_Masters_all/burtonml.xml [hunk 1]
```diff
@@ -133,7 +133,7 @@
         <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no98115213">Babst, Earl D., 1870-1967.</persname>
         <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/n50017666">Balfour, Arthur James, 1848-1930.</persname>
         <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no2001090656">Bates, Henry Moore, 1869-1949.</persname>
-        <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no2008186663">Beal, Junius E. (Junius Emery), 1860-1942.</persname>
+        <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no2008186663">Beal, Junius Emery, 1860-1942.</persname>
```
Comment (eckardm) [inline Real_Masters_all/burtonml.xml]: Hi Walker,

I apologize, I should have given you some more context. In this case, you've picked the older format. Here's the new format:

http://lccn.loc.gov/n94104489
Authorized (i.e., we want this one): Damon, Robert H. (Robert Harris), 1908-
Variant (i.e., the old way): Nutt, Damon Harris, 1908-

Yeah, sorry about that. Dallas and I are in a meeting now but we can talk about this after.

File: Real_Masters_all/aagovp.xml [hunk 2]
```diff
@@ -38,7 +38,7 @@
   <archdesc level="recordgrp" audience="external" type="inventory" relatedencoding="MARC21">
     <did>
       <origination>
-        <corpname source="lcnaf" encodinganalog="110">Ann Arbor (Mich.)</corpname>
+        <geogname source="lcnaf" encodinganalog="110">Ann Arbor (Mich.)</geogname>
```
Comment (djpillen) [inline Real_Masters_all/aagovp.xml]: I think this should maybe stay as a corpname -- In this case, it is not the geographic location of Ann Arbor that is the creator of the Ann Arbor government publications, but the corporate entity of Ann Arbor. Subjects cannot be associated with resources as creators, only agents.

Comment (walkerdb) [inline Real_Masters_all/aagovp.xml]: ah - I'll make the change

File: Real_Masters_all/burtonml.xml (hunk 1 above)
Comment (walkerdb) [inline Real_Masters_all/burtonml.xml]: ok - should be an easy fix

**Comment by walkerdb** [comment]
Things should be in the preferred normal form now

**Comment by djpillen** [comment]
This looks mostly right. The only issue I caught is that the Polar Bear Expedition is actually a corporate body, not a subject. That should be an easy fix though.

**Comment by walkerdb** [comment]
Really? I went with subject originally because the majority of the Polar Bear Expedition terms (there are ~110 of them) were subject tags. Should they all be changed over to corpname?

**Comment by djpillen** [comment]
Hm.. this could be another thing like Ann Arbor (Mich.) where it is both a geographic term and a corporate body. I bet that there are some finding aids that use Polar Bear Expedition as a related subject and to refer to the corporate body. I dunno... maybe it would be best to leave them as they are and assume that the catalogers at the time used them differently on purpose.

**Comment by walkerdb** [comment]
Ok - I think the polar-bear commit should be removed now.

**Comment by eckardm** [comment]
Could you also run this on without-reservations! Thank for all this BTW.

**Comment by walkerdb** [comment]
should in without-reservations now

**Pull request merged**